modules/eeprom_24xx64.c

```diff
@@ -89,7 +89,7 @@ size_t eeprom_24xx64_write( uint8_t id, uint16_t address, uint8_t *tx_data, size
 
             /* Write the data */
             tx_len += xI2CMasterWrite( i2c_interface, i2c_addr, &page_buf[0] , bytes_to_write+2 );
-            vTaskDelay(10);
+            vTaskDelay(pdMS_TO_TICKS(1));
```

Comment:
You are changing the delay here from 10 ms to 1 ms. This is correct, but it is unrelated to the scope of this PR. Please, keep it at 10 ms for now.